python/GafferSceneUI/HierarchyView.py

```diff
@@ -100,6 +100,17 @@ def scene( self ) :
 
 		return self.__plug
 
+	## Returns the widget used for showing the main scene listing, with the
+	# intention that clients can add custom context menu items via
+	# `sceneListing.pathColumnContextMenuSignal()`.
+	#
+	# > Caution : This currently returns a PathListingWidget, but in future
+	# > will probably return a more specialised widget with fewer privileges.
+	# > Please limit usage to `pathColumnContextMenuSignal()`.
```

Comment:
Should these be referring to `columnContextMenuSignal()` rather than `pathColumnContextMenuSignal()`?